### PR TITLE
more updates to mergedoptions

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: tests/Microsoft.Identity.Web.Test/MicrosoftIdentityOptionsTests.cs [hunk 1]
```diff
@@ -14,6 +19,9 @@ public class MicrosoftIdentityOptionsTests
         private MicrosoftIdentityOptions _microsoftIdentityOptions;
```
Comment (jmprieur) [inline tests/Microsoft.Identity.Web.Test/MicrosoftIdentityOptionsTests.cs:19]:
```suggestion
```

File: tests/Microsoft.Identity.Web.Test/MicrosoftIdentityOptionsTests.cs [hunk 2]
```diff
@@ -14,6 +19,9 @@ public class MicrosoftIdentityOptionsTests
...
         private const string AzureAd = Constants.AzureAd;
         private const string AzureAdB2C = Constants.AzureAdB2C;
+        private ServiceProvider _provider;
+        private IOptionsMonitor<MicrosoftIdentityOptions> _microsoftIdentityOptionsMonitor;
+        private IOptionsMonitor<ConfidentialClientApplicationOptions> _applicationOptionsMonitor;
```
Comment (jmprieur) [inline tests/Microsoft.Identity.Web.Test/MicrosoftIdentityOptionsTests.cs:24]: Is it needed?

**Review by jmprieur — APPROVED** [review]
LGTM.

File: tests/Microsoft.Identity.Web.Test/MicrosoftIdentityOptionsTests.cs (hunk 2 above)
Comment (jennyf19) [reply tests/Microsoft.Identity.Web.Test/MicrosoftIdentityOptionsTests.cs:24]: these are yes.

File: tests/Microsoft.Identity.Web.Test/MicrosoftIdentityOptionsTests.cs (hunk 1 above)
Comment (jennyf19) [reply tests/Microsoft.Identity.Web.Test/MicrosoftIdentityOptionsTests.cs:19]: removed.

**Pull request merged**